.github/workflows/publish-technical-documentation-next.yml

```
@@ -0,0 +1,42 @@
+name: "publish-technical-documentation-next"
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "docs/sources/**"
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out code"
+        uses: "actions/checkout@v3"
+      - name: "Build website"
+        # -e HUGO_REFLINKSERRORLEVEL=ERROR prevents merging broken refs with the downside
+        # that no refs to external content can be used as these refs will not resolve in the
+        # docs-base image.
+        run: |
+          docker run -v ${PWD}/docs/sources:/hugo/content/docs/loki/latest -e HUGO_REFLINKSERRORLEVEL=ERROR --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
```

Very small nit, but since this is related to the next directory, we can also update the mapping docker directory:
```suggestion
          docker run -v ${PWD}/docs/sources:/hugo/content/docs/loki/next -e HUGO_REFLINKSERRORLEVEL=ERROR --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
```